src/components/controls/mixins.scss

```diff
@@ -3,28 +3,32 @@
 @use './variables.scss' as control-variables;
 
 @mixin input-control($size) {
+    --_--input-control-border-width: var(
+        --g-text-input-border-width,
```

Comment:
This mixin is also used in TextArea component, we can't rely only on TextInput's var